### PR TITLE
Message contents should be checked on null before being reused

### LINKED ISSUE #14: [bug] RangeError on messageDelete
```
(node:30) UnhandledPromiseRejectionWarning: RangeError [EMBED_FIELD_VALUE]: MessageEmbed field values may not be empty.
    at Function.normalizeField (/usr/src/app/node_modules/discord.js/src/structures/MessageEmbed.js:425:23)
    at /usr/src/app/node_modules/discord.js/src/structures/MessageEmbed.js:445:14
    at Array.map (<anonymous>)
    at Function.normalizeFields (/usr/src/app/node_modules/discord.js/src/structures/MessageEmbed.js:444:8)
    at MessageEmbed.addFields (/usr/src/app/node_modules/discord.js/src/structures/MessageEmbed.js:259:42)
    at MessageEmbed.addField (/usr/src/app/node_modules/discord.js/src/structures/MessageEmbed.js:250:17)
    at /usr/src/app/dist/events/messageDelete.js:50:14
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/dist/events/messageDelete.js:5:58)
    at runMicrotasks (<anonymous>)
```

### PR DESCRIPTION
**Pull request by notfooddev**

### CODE REVIEW

File: src/events/messageUpdate.ts [hunk 1]
```diff
@@ -34,10 +34,14 @@ export default async function MessageUpdateEvent(discordClient: DiscordClient, o
...
-		.addField("Channel", `${oldMessage.channel}`, true)
-		.addField("Before", `${oldMessage.content}`, false)
-		.addField("After", `${newMessage.content}`, false)
-		.setFooter(`User ID: ${newMessage.author.id}`);
+		.addField("Channel", `${oldMessage.channel}`, true);
+	
+	if (oldMessage.content) { 
+		embed.addField("Before", `${oldMessage.content}`, false)
+	}
+	embed.addField("After", `${newMessage.content}`, false);
```
Comment (WesVleuten) [inline src/events/messageUpdate.ts]: `newMessage.content` can be empty

**Review by WesVleuten — CHANGES_REQUESTED** [review]

File: src/events/messageUpdate.ts (hunk 1 above)
Comment (notfooddev) [reply src/events/messageUpdate.ts]: You can't change a message to have empty content, you simply delete the message in that case.

File: src/wrapper/discord/messageEmbed.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	fields?: Field[];
+}
+
+function randomColor() {
+	return "#000000".replace(/0/g, () => (~~(Math.random() * 16)).toString(16));
+}
+
+function safeFieldValue(input: string): string {
+	if (input == null || input == '') {
+		return "**Empty String**";
```
Comment (notfooddev) [inline src/wrapper/discord/messageEmbed.ts]: String is too technical towards user

File: src/wrapper/discord/messageEmbed.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	}
+	return input;
+}
+
+export default function createEmbed(input: Embed): MessageEmbed {
```
Comment (notfooddev) [inline src/wrapper/discord/messageEmbed.ts]: Are there any kind of other embeds in the discord.js library? Should we call this createMessageEmbed instead perhaps?

**Pull request merged**